scipy/signal/windows/windows.py

```diff
@@ -816,7 +816,13 @@ def hann(M, sym=True):
     return _truncate(w, needs_trunc)
 
 
-hanning = hann
+def hanning(*args, **kwargs):
+    """
+    Alias of `hann`
+
+    This is for backwards compatibility. See `hann` for more details.
+    """
+    hann(*args, **kwargs)
```

Comment:
Missing **`return`** statement.  This line should be

    return hann(*args, **kwargs)

Reply:
derp. I'll fix that.